system-addon/content-src/styles/activity-stream-linux.scss

```diff
@@ -1,2 +1,20 @@
 /* This is the linux variant */
 @import './activity-stream';
+
+// Custom image and size of the arrow from the (?) panel that shows more info
+// about a section.
+.section-info-option .info-option {
+  $arrow-height: 10px;
```

Comment:
Duplicating these 3x and away from where the rest of it's defined is going to make this quite hard to maintain. I think we should have a different pattern of something like:
```css
$os-info-arrow-height: 10px;
…
@import './activity-stream'; 
```

Reply:
@Mardak if the platform-specific-css is to go where the `...` are then they will not be able to take advantage of the variables defined in _variables.scss.

Reply:
I guess we can import them again at the top of each platform specific file.